### PR TITLE
feature/RA-improvement

### PR DESCRIPTION
**Pull request by ShilpiMaurya**
# Changes done
- Improved the design of the website.

### CODE REVIEW

File: src/resume.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "basics": {
     "name": "Shilpi Maurya",
-    "age": "24 Years Old",
-    "label": "Aspiring Front-end Developer",
+    "age": "25 years old",
+    "label": "Aspiring front-end developer",
```
Comment (vaibhav-kumar-tokyo) [inline src/resume.json:5]: can remove `Aspiring`, you are a front-end dev now :)

File: src/resume.json [hunk 2]
```diff
@@ -21,20 +21,39 @@
...
+        "courses": "js basics, js behind the scenes, advanced js: objects and function, next generation js"
+      }
+    ],
+
+    [
+      {
+        "institution": "Udemy (online course)",
+        "area": "JavaScript algorithms and data structures masterclass",
+        "startDate": "2020 / January",
+        "endDate": "on going",
```
Comment (vaibhav-kumar-tokyo) [inline src/resume.json:37]: `ongoing` is a single word, right?

File: src/resume.json [hunk 3]
```diff
@@ -21,20 +21,39 @@
...
     ],
     [
       {
         "institution": "Inderprastha Engineering College, India",
         "area": "Electrical and Electronics Engineering (Bachelor)",
-        "startDate": "2013 / July",
+        "startDate": "2013 / August",
         "endDate": "2017 / June",
-        "courses": "Electrical machines - Switch gear and protection - Human Values and Professional Ethics - Cyber Security"
+        "courses": "electrical machines, switch gear and protection, human Values and professional ethics, cyber security"
```
Comment (vaibhav-kumar-tokyo) [inline src/resume.json:56]: Values -> values

Comment (vaibhav-kumar-tokyo) [reply src/resume.json:56]: can add any programming or electronic courses here, as they contribute to your dev skills or technical thinking or problem solving

**Comment by vaibhav-kumar-tokyo** [comment]
image
padding missing

**Comment by vaibhav-kumar-tokyo** [comment]
Can not really ready some skills:
image

even when we have space in mobile view
image

**Comment by vaibhav-kumar-tokyo** [comment]
> Can not really ready some skills:
> image
> 
> even when we have space in mobile view
> image

Do you wanna try putting skills in pills and remove the skill-level (Intermediate, Beginner etc.)?

**Comment by ShilpiMaurya** [comment]
# Skill section in pill view
Screen Shot 2020-06-11 at 21 10 36

**Comment by ShilpiMaurya** [comment]
# Skill section in pill view
Screen Shot 2020-06-11 at 21 31 52

**Comment by vaibhav-kumar-tokyo** [comment]
Pls remove Eslint and prettier and  CSS animation

**Comment by vaibhav-kumar-tokyo** [comment]
> Pls remove Eslint and prettier and CSS animation

instead you could write `next.js` and `gatsby`

**Comment by vaibhav-kumar-tokyo** [comment]
Also, profile image can be changed to one you are using on medium, it looks more professional.

File: src/Navigation/Skill/index.css [hunk 4]
```diff
@@ -53,12 +53,18 @@
 .skills_web-development_level {
   background-color: var(--primary-color);
   color: var(--button-color);
-  border-radius: 10px;
+  border-radius: 20px;
   text-align: center;
   align-items: center;
   align-content: center;
-  font-size: 18px;
+  /* font-size: 18px; */
```
Comment (vaibhav-kumar-tokyo) [inline src/Navigation/Skill/index.css:60]: remove the comment

**Comment by vaibhav-kumar-tokyo** [comment]
Font size of skill-pills seem too big.

**Comment by ShilpiMaurya** [comment]
# Modified skill section

## With different screen sizes

Screen Shot 2020-06-11 at 23 29 05

Screen Shot 2020-06-11 at 23 29 27

**Comment by vaibhav-kumar-tokyo** [comment]
Two more changes:
1. Too much padding on pills.
2. skills section has uneven spacing in start and end, as shown here
image

**Comment by vaibhav-kumar-tokyo** [comment]
> image
> padding missing

this is not fixed yet

**Comment by vaibhav-kumar-tokyo** [comment]
[Suggestion] Regarding pills width, let's try giving them a `min-width` instead of `width` and see if it looks better?

**Comment by ShilpiMaurya** [comment]
# Improvement in Pills design

## With different screen sizes

Screen Shot 2020-06-12 at 17 41 44

Screen Shot 2020-06-12 at 17 42 42

**Comment by ShilpiMaurya** [comment]
> > image
> > padding missing
> 
> this is not fixed yet

>>I didn't understand this comment whether  padding around whole nav bar or individual blocks.

**Comment by vaibhav-kumar-tokyo** [comment]
> > > image
> > > padding missing
> > 
> > 
> > this is not fixed yet
> 
> > > I didn't understand this comment whether  padding around whole nav bar or individual blocks.

This whole section, in mobile view the text hugs the viewport's boundary.

File: src/Navigation/Skill/index.css [hunk 5]
```diff
@@ -53,12 +53,17 @@
...
   text-align: center;
   align-items: center;
   align-content: center;
-  font-size: 18px;
+  font-size: 22px;
   justify-content: center;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+  padding: 5px;
```
Comment (vaibhav-kumar-tokyo) [inline src/Navigation/Skill/index.css:65]:
```suggestion
  padding: 6px 12px;
```

File: src/Navigation/Skill/index.css [hunk 6]
```diff
@@ -78,6 +83,10 @@
   justify-content: center;
   align-content: center;
   align-items: center;
+  transition: all 0.5s ease-in-out;
```
Comment (vaibhav-kumar-tokyo) [inline src/Navigation/Skill/index.css:86]: on line 81: change padding to 12px

**Comment by vaibhav-kumar-tokyo** [comment]
Also, pls check your site on safari

**Comment by ShilpiMaurya** [comment]
# Latest look of Skill section and nav bar
## Changes done
- Adjusted padding of pills
- Gave pills some shadow

Screen Shot 2020-06-13 at 18 36 18

## Changes done
- Adjusted size of nav-bars

Screen Shot 2020-06-13 at 18 36 02

Screen Shot 2020-06-13 at 18 54 54

**Comment by ShilpiMaurya** [comment]
> Also, pls check your site on safari
>Have to fix this problem

Screen Shot 2020-06-13 at 18 58 15

**Comment by vaibhav-kumar-tokyo** [comment]
please commit your changes so that I can check

**Comment by vaibhav-kumar-tokyo** [comment]
> > > > image
> > > > padding missing
> > > 
> > > 
> > > this is not fixed yet
> > 
> > 
> > > > I didn't understand this comment whether  padding around whole nav bar or individual blocks.
> 
> This whole section, in mobile view the text hugs the viewport's boundary.

also, did you fix this issue?

**Comment by ShilpiMaurya** [comment]
> > > > > image
> > > > > padding missing
> > > > 
> > > > 
> > > > this is not fixed yet
> > > 
> > > 
> > > > > I didn't understand this comment whether  padding around whole nav bar or individual blocks.
> > 
> > 
> > This whole section, in mobile view the text hugs the viewport's boundary.
> 
> also, did you fix this issue?

Screen Shot 2020-06-13 at 19 25 48

File: src/Navigation/NavBar/index.css [hunk 7]
```diff
@@ -8,25 +8,35 @@
...
-  height: 25px;
-  font-size: 15px;
+  /* width: 85px;
+  height: 25px; */
+  font-size: 20px;
   border-radius: 100px;
   font-weight: 400;
   outline: none;
+  -moz-box-shawdow: var(--shadow-style);
+  -webkit-box-shadow: var(--shadow-style);
```
Comment (vaibhav-kumar-tokyo) [inline src/Navigation/NavBar/index.css:26]: can try using drop-shadow filter too
https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/drop-shadow

**Pull request merged**